clusters/threat-actor.json

```diff
@@ -6511,20 +6521,6 @@
       "uuid": "d8e1762a-0063-48c2-9ea1-8d176d14b70f",
       "value": "STARDUST CHOLLIMA"
     },
-    {
-      "description": "In short, “Cold River” is a sophisticated threat (actor) that utilizes DNS subdomain hijacking, certificate spoofing, and covert tunneled command and control traffic in combination with complex and convincing lure documents and custom implants.",
-      "meta": {
-        "refs": [
-          "https://www.lastline.com/labsblog/threat-actor-cold-river-network-traffic-analysis-and-a-deep-dive-on-agent-drable/"
```

Comment:
Dead link, can't find it anywhere

Reply:
There is a copy on archive.org -> https://web.archive.org/web/20220618182654/https://www.lastline.com/labsblog/threat-actor-cold-river-network-traffic-analysis-and-a-deep-dive-on-agent-drable/